### PR TITLE
documentation: Update README.md

### LINKED ISSUE #8: docs: Update readme with instructions to setup and run the app in both local and production environments

### PR DESCRIPTION
**Pull request by wal33d006**
Added instructions on how to setup and run the app on local and hosted environments.

Fixes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# flutter_app
+# Flutter application for integrating with Cosmos-SDK
```
Comment (fadeev) [inline README.md:1]:
```suggestion
# Flutter app for integrating with Cosmos SDK
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-A new Flutter project.
+This is a sample application which is integrated to Cosmos-SDK for adding wallets through mnemonnics and sending tokens to different addresses.
```
Comment (fadeev) [inline README.md:3]:
```suggestion
This is a sample application for a Cosmos SDK blockchain that supports basic wallet management (importing mnemonics), and sending tokens.
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-## Getting Started
+## Setup (Localhost)
```
Comment (fadeev) [inline README.md:5]:
```suggestion
## Setup (on `localhost`)
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-This project is a starting point for a Flutter application.
+- [Setup Flutter](https://flutter.dev/docs/get-started/install)
+- [Install Starport](https://docs.starport.network/intro/install.html)
+- Scaffold a blockchain using `starport app github.com/hello/planet`
+- Navigate to `planet` folder and run `starport serve`
+- After it is successfully built, you will see two mnenomics for Alice and Bob along with their wallet addresses in the terminal
```
Comment (fadeev) [inline README.md:11]:
```suggestion
- After it is successfully built, you will see two mnemonics for Alice and Bob along with their wallet addresses in the terminal
```

File: README.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+- Run your Flutter app without any extra command line arguments and add both wallets in your app using the mnemonics from the terminal
```
Comment (fadeev) [inline README.md:12]: How do I run a Flutter app?

File: README.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
 
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
-
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+## Setup (Hosted Testnet)
```
Comment (fadeev) [inline README.md:14]:
```suggestion
## Setup (Cosmos Hub Testnet)
```

**Review by fadeev — CHANGES_REQUESTED** [review]
Looks good! Made a few minor adjustments. I would also add info on how to tun the Flutter apps for those who are not familiar. That'd be very helpful!

File: README.md (hunk 5 above)
Comment (wal33d006) [reply README.md:12]: The Flutter setup link describes everything in detail regarding setting up and running Flutter app. I will add separate steps as well

**Comment by wal33d006** [comment]
@fadeev Just updated the doc

**Review by fadeev — APPROVED** [review]
💯

**Pull request merged**